plugin/plugin.xml

```diff
@@ -51,6 +51,11 @@
       id="io.snyk.eclipse.plugin.commands.execute"
       name="%command.name">
     </command>
+    <command
```

Comment:
Didn't we say that the wizard should be folded into the Snyk scan command instead of a second menu option?

Reply:
Also, the text of the menu entry should change IIRC, `Configure Snyk` instead of `Snyk Scan` when token is set. This can be achieved with a condition here.

Reply:
We spoke to Annalisa who mentioned that a static `Snyk Test` over a dynamic `Configure Snyk / Snyk Test` was a better user experience.